lib/vec.c

```diff
@@ -739,9 +743,37 @@ M0_INTERNAL bool m0_ivec_cursor_move_to(struct m0_ivec_cursor *cur,
 		if (ret)
 			break;
 	}
+
 	return ret;
 }
 
+M0_INTERNAL m0_bindex_t m0_ivec_cursor_conti(const struct m0_ivec_cursor *cur,
+					     m0_bindex_t dest)
+{
+	uint32_t i;
+	m0_bindex_t idx;
+	m0_bindex_t max_seg_end;
+	struct m0_indexvec *ivec;
```

Comment:
[Style] alignment of variables is needed.

Reply:
Fixed, thanks.